docs/using/installing/index.rst

```diff
@@ -4,17 +4,27 @@
 Installing Flocker
 ==================
 
-As a user of Flocker you will need to install the ``flocker-cli`` package which provides command line tools to control the cluster.
-This should be installed on a machine with SSH credentials to control the cluster nodes
-(e.g., if you use our Vagrant setup then the machine which is running Vagrant).
+The following diagram illustrates the architecture of the cluster you are about to install:
 
-There is also a ``clusterhq-flocker-node`` package which is installed on each node in the cluster.
-It contains the services that need to run on each node.
+.. image:: images/installintro.png
+   :width: 75%
+   :alt: Diagram illustrating the Flocker CLI and control service and agents.
```

Comment:
This is not suitable alt text. Alternative text should provide the reader with the information provided by the image. There are many conflicting views on exactly what is best practice for alt text, but the RNIB link to http://webaim.org/techniques/alttext/#context for example. Try to put yourself in the position of someone who cannot see the image - that person would preferably have all of the information that a fully sighted person has at the end of reading this document. If everything is explained in the text already then there may be no need for alt text.